template.json

```diff
@@ -10,39 +10,35 @@
       "prepare": "husky install"
     },
     "dependencies": {
-      "@testing-library/jest-dom": "^5.14.1",
-      "@testing-library/react": "^13.0.1",
-      "@testing-library/user-event": "^13.2.1",
-      "@types/jest": "^27.0.1",
-      "@types/node": "^16.7.13",
-      "@types/react": "^17.0.20",
-      "@types/react-dom": "^17.0.9",
-      "typescript": "^4.4.2",
-      "web-vitals": "^2.1.0",
-      "classnames": "^2.3.1",
+      "classnames": "^2.3.2",
       "env-cmd": "^10.1.0",
-      "react-router-dom": "^6.2.1",
-      "@hipo/react-ui-toolkit": "^1.0.0-beta"
+      "react-router-dom": "^6.14.2",
+      "@hipo/react-ui-toolkit": "^1.1.1"
     },
     "devDependencies": {
-      "@hipo/eslint-config-base": "^4.2.0",
-      "@hipo/eslint-config-react": "^2.2.0",
-      "@hipo/eslint-config-typescript": "^1.1.0",
-      "@hipo/stylelint-config-base": "^2.2.0",
-      "@types/react-router-dom": "^5.3.1",
-      "eslint-config-prettier": "^8.3.0",
+      "@hipo/eslint-config-base": "^4.3.0",
+      "@hipo/eslint-config-react": "^2.3.0",
+      "@hipo/eslint-config-typescript": "^1.2.0",
+      "@hipo/stylelint-config-base": "4.0.0",
+      "@types/react-router-dom": "^5.3.3",
+      "eslint-config-prettier": "^8.6.0",
       "eslint-import-resolver-typescript": "^2.5.0",
-      "husky": "^7.0.2",
-      "lint-staged": "^11.2.0",
-      "node-sass": "^6.0.1",
-      "prettier": "^2.4.1",
-      "stylelint": "^13.13.1",
-      "stylelint-no-unsupported-browser-features": "^5.0.2",
-      "stylelint-order": "^4.1.0",
-      "stylelint-scss": "^3.21.0"
+      "husky": "^8.0.3",
+      "lint-staged": "^13.1.2",
+      "sass": "^1.63.6",
+      "typescript": "^4",
+      "prettier": "^2.8.4",
+      "stylelint": "^15.2.0",
+      "stylelint-no-unsupported-browser-features": "^6.1.0",
+      "stylelint-order": "^6.0.2",
+      "stylelint-scss": "^4.4.0",
+      "@types/react": "*",
+      "@types/react-dom": "*"
```

Comment:
These are `*` since we don't specify react version, and it installs the latest version

Reply:
Would "latest" work? It might be more descriptive

Reply:
yeah I think that's more suitable for this case, updated